nixos/tests/invidious.nix

```diff
@@ -5,48 +5,72 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     maintainers = [ sbruder ];
   };
 
-  nodes.machine = { config, lib, pkgs, ... }: {
-    services.invidious = {
-      enable = true;
+  nodes = {
+    postgres-tcp = { config, pkgs, ... }: {
+      services.postgresql = {
+        enable = true;
+        initialScript = pkgs.writeText "init-postgres-with-password" ''
+          CREATE USER invidious WITH PASSWORD 'correct horse battery staple';
+          CREATE DATABASE invidious WITH OWNER invidious;
```

Comment:
Not sure if this could use ensureDBOwnership

Reply:
This is an... Explicit TCP/IP test?

Reply:
I'm not really sure why we're testing PostgreSQL over TCP to be honest. I think the test was originally meant to test the password file which requires TCP instead of a unix socket.
I moved this to a separate VM so the test runs a bit faster as the secondary VM can start PostgreSQL while the first VM runs the different nginx tests.